Specification.md

```diff
@@ -0,0 +1,87 @@
+# XApp Specification
+
+TBD: I suppose it would be nice to call the project XApp instead of xapp ? (used lower case for now)
+
+In order to be shown on the xapp project, applications need to fulfill a list of requirements. This specification is used in order to summarize these requirements and as such can be utilized to decide if an application can join the xapp project.
+
+## Requirements
+
+### Licensing
+
+Xapps need to be licensed on an [OSI-approved license](https://opensource.org/licenses).
+
+### Operating System
+
+Xapps need to be able to run in a Linux environment. The support of other operating systems is optional.
```

Comment:
This feels a little bit too negative toward other OSes.  I think it's fine that Linux is the only requirement, but it might be friendlier to include something like, "Support for other operating systems is optional, but application developers should strive to write portable code where possible."

Reply:
Hmm, not sure for this one. It sounds a bit like xapp devs should know how to write code which is good to port across OSes.

Maybe like this to get the phrase less negative: "Support for other operating systems is welcome, though optional." ?

Reply:
FIxed: 0f7836e9970d867d1ab141c3f9f055e68fa32fd0